scripts/upload-theme.sh

```
@@ -0,0 +1,334 @@
+set -e
+
+THEME_ID="${1:-nomad}"
+BUCKET="karaoke-gen-storage-nomadkaraoke"
+THEMES_PATH="themes"
+BRANDING_DIR="/Users/andrew/AB Dropbox/Andrew Beveridge/MediaUnsynced/Karaoke/NomadBranding"
+
+echo "=== Uploading theme: $THEME_ID to gs://$BUCKET/$THEMES_PATH/ ==="
+
+# Create temporary directory for processing
+TEMP_DIR=$(mktemp -d)
+trap "rm -rf $TEMP_DIR" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)